_includes/footer.html

```diff
@@ -4,14 +4,18 @@
   CC BY NC 4.0</a>
   </p>
 
-  <p>Copyright 2009-2015 Software Freedom Conservancy, Sugar Labs®</p>
+  <p id="copy"></p>
+<script>
+  document.getElementById("copy").innerHTML="Copyright &copy; 2009-"+new Date().getFullYear()+ " Software Freedom Conservancy, Sugar Labs&reg;";
```

Comment:
A copyright notice isn't required for copyright to adhere in a work since the Berne convention. Its nice to include the year, but not necessary :)